### ISSUE TITLE
Wish to get the detail information about the pretrained diffusion model on ffhq-256 dataset

### ISSUE THREAD

**Issue by cuttle-fish-my** [yandex-research/ddpm-segmentation#20]
Hi, I'm just reproducing the results of your paper. For some reason, we have to train ourselves diffusion model on ffhq-256 dataset. However, even using the modal flag in train_intepreter.py, the kmeans results of our diffusion model are always more coarser than pretrained model's. So I want more detailed training information such as batch size and training epoch, etc. Thanks!

**Comment by dbaranchuk** [comment]
Hey,

The exact hyperparameter list on 8x gpus:
 
TRAIN_FLAGS="--lr 5e-5 --batch_size 16 --seed 42 --save_interval 4000"
MODEL_FLAGS="--attention_resolutions 32,16,8 --class_cond False --diffusion_steps 1000 --dropout 0.1 --image_size 256 --learn_sigma True --noise_schedule linear --num_channels 256 --num_head_channels 64 --num_res_blocks 2 --resblock_updown True --use_fp16 True --use_scale_shift_norm True"

Note that batch_size=16 per 1 GPU --> the overall batch size is 128. Regarding the nb of epochs, unfortunately, I do not remember.
